Tests/Docker/Container/NetworkTest.php

```diff
@@ -22,7 +22,7 @@ private function getBuilderImageConfiguration()
                             'uri' => 'https://github.com/keboola/docker-demo-app', // not used, can by anything
                             'type' => 'git',
                         ],
-                        'entry_point' => 'ping -W 10 -c 1 www.example.com',
+                        'entry_point' => 'if ping -W 10 -c 1 www.example.com; then return 0; else return 1; fi',
```

Comment:
jak se sem tohle dostalo?

Reply:
pokud je to z nějakýho updatovanýho docker image, tak bych v tomhle PR zafixoval jeho verzi na starší a tohle hodil v samostatným PR.